### ISSUE TITLE
[REQUEST] DRAWLINEHUMAN

### ISSUE THREAD

**Issue by Jolge2** [openbullet/openbullet#335]
i need a little help
is there any way for put random number  for this mouse action 
```
MOUSEACTION
  DRAWLINEHUMAN a b-> c d : 150 1 1
```
a , b , c , d 

i made random number with php and used Parse block like this 

```
MOUSEACTION
  DRAWLINEHUMAN <random1> <random2>-> <random3> <random4>: 150 1 1
```

but doesn't work

**Comment by openbullet** [comment]
Hey, there is a way yes.

First you have to create 4 random numbers using function blocks, then you have to spawn 2 elements at those coordinates on the page using the EXECUTEJS block (variables like <VAR> will be replaced before the script is executed) and then reference those elements by ID.

I will keep this issue open so I can rework it as it takes too much work right now.

If you want the mouse to just move randomly very quickly on the screen though you might as well use the draw points function.

**Comment by Jolge2** [comment]
i want click mouse randomly for exmple in square with this coordinates  400 400
i a little noobie in this work
Thank you very much if you write a example with numbers here  
i tried but cant do this
thanks

**Comment by Jolge2** [comment]
it doesnt read 
always says 
ERROR: Expected Integer value for 'OFFSET X'

**Comment by openbullet** [comment]
If you do it exactly as I say, it works. If you do it as you typed it in, it will never work unless I change the code.

**Comment by Jolge2** [comment]
#MW1 FUNCTION RandomNum 1 450 -> VAR "MW1" 

#MW2 FUNCTION RandomNum 1 450 -> VAR "MW2" 

MOUSEACTION
MOVEBY <MW1> <MH1>
CLICK

i did this

**Comment by openbullet** [comment]
You can use moveby with integers, but you cannot use drawlinehuman with the same method, I did not implement the support for it. So please just wait for the implementation okay?

**Comment by Jolge2** [comment]
THANKS

**Comment by openbullet** [comment]
Implemented in 1.2.0
